app/src/main/res/values/strings.xml

```diff
@@ -721,4 +721,11 @@ be temporary, and you can try again later.</li>
     <!-- Tip displayed on home view explaining how to add a custom autocomplete URL
         Argument 1 is the app name (Firefox Focus/Firefox Klar/etc). -->
     <string name="tip_disable_tips">Turn off tips on the %1$s start screen</string>
+
+    <!-- Displayed in a snackbar when a user opens a new tab, gives action to switch to the newly opened tab -->
+    <string name="new_tab_opened_snackbar">New Tab Opened</string>
```

Comment:
Small nitpick, but shouldn't this be "New tab opened"?

Reply:
Oops thanks for the catch!